sdk/servicebus/service-bus/test/utils/aadUtils.ts

```diff
@@ -1,117 +1,21 @@
-import { ServiceBusManagementClient } from "@azure/arm-servicebus";
-import { SBQueue, SBTopic, SBSubscription } from "@azure/arm-servicebus/esm/models";
-import { EnvVarKeys, getEnvVars } from "./envVarUtils";
-
-const _loginWithServicePrincipalSecret = msRestNodeAuth.loginWithServicePrincipalSecret;
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
 
-export const loginWithServicePrincipalSecret = _loginWithServicePrincipalSecret;
+import { EnvVarKeys, getEnvVars } from "./envVarUtils";
 
-export async function recreateQueue(queueName: string, parameters: SBQueue): Promise<void> {
-  const env = getEnvVars();
-  await msRestNodeAuth
-    .loginWithServicePrincipalSecret(
-      env[EnvVarKeys.AAD_CLIENT_ID],
-      env[EnvVarKeys.AAD_CLIENT_SECRET],
-      env[EnvVarKeys.AAD_TENANT_ID]
-    )
-    .then(async (creds) => {
-      const client = await new ServiceBusManagementClient(
-        creds,
-        env[EnvVarKeys.AZURE_SUBSCRIPTION_ID]
-      );
-      await client.queues.deleteMethod(
-        env[EnvVarKeys.RESOURCE_GROUP],
-        getNamespace(env[EnvVarKeys.SERVICEBUS_CONNECTION_STRING]),
-        queueName,
-        function(error: any): void {
-          if (error) throw error.message;
-        }
-      );
-      await client.queues.createOrUpdate(
-        env[EnvVarKeys.RESOURCE_GROUP],
-        getNamespace(env[EnvVarKeys.SERVICEBUS_CONNECTION_STRING]),
-        queueName,
-        parameters,
-        function(error: any): void {
-          if (error) throw error.message;
-        }
-      );
-    });
-}
+import { loginWithServicePrincipalSecret } from "@azure/ms-rest-nodeauth";
 
-export async function recreateTopic(topicName: string, parameters: SBTopic): Promise<void> {
-  const env = getEnvVars();
-  await msRestNodeAuth
-    .loginWithServicePrincipalSecret(
-      env[EnvVarKeys.AAD_CLIENT_ID],
-      env[EnvVarKeys.AAD_CLIENT_SECRET],
-      env[EnvVarKeys.AAD_TENANT_ID]
-    )
-    .then(async (creds) => {
-      const client = await new ServiceBusManagementClient(
-        creds,
-        env[EnvVarKeys.AZURE_SUBSCRIPTION_ID]
-      );
-      await client.topics.deleteMethod(
-        env[EnvVarKeys.RESOURCE_GROUP],
-        getNamespace(env[EnvVarKeys.SERVICEBUS_CONNECTION_STRING]),
-        topicName,
-        function(error: any): void {
-          if (error) throw error.message;
-        }
-      );
-      await client.topics.createOrUpdate(
-        env[EnvVarKeys.RESOURCE_GROUP],
-        getNamespace(env[EnvVarKeys.SERVICEBUS_CONNECTION_STRING]),
-        topicName,
-        parameters,
-        function(error: any): void {
-          if (error) throw error.message;
-        }
-      );
-    });
-}
+const aadServiceBusAudience = "https://servicebus.azure.net/";
 
-export async function recreateSubscription(
-  topicName: string,
-  subscriptionName: string,
-  parameters: SBSubscription
-): Promise<void> {
+export const getTokenCredentialsFromAAD = async () => {
```

Comment:
For functions, we have generally used the pattern
`export async function functionName() {}`

Any reason to deviate and use the `export const functionName = async() => {}` patterns?

Reply:
Previously we used this format as can be seen in aadUtils.browser.ts
Updated to use the proposed general pattern